packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/flush-archive/translations/Messages_fr_FR.json

```diff
@@ -1,12 +1,12 @@
 {
-  "pci_projects_project_storages_containers_container_cold_archive_flush_title": "Voulez-vous vider votre conteneur ?",
-  "pci_projects_project_storages_containers_container_cold_archive_flush_description": "Vous êtes sur le point de vider le conteneur: {{ containerName }} de ses objects.",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_title": "Voulez-vous vider votre archive ?",
+  "pci_projects_project_storages_containers_container_cold_archive_flush_description": "Vous êtes sur le point de vider l'archive: {{ containerName }} de ses objects.",
```

Comment:
**question**: should we not use the french version of `objects` here ?

Reply:
You're right, there is a typo here.